test/integration/routes.test.js

```diff
@@ -155,6 +155,27 @@ describe('routes', function () {
         expect(getItemResult.body.metadata_keys).to.deep.equal(['testFile'])
       })
 
+      test('add item that consumes a parent', async function () {
+        // add parent to be consumed
+        const firstToken = await postRunProcess(app, authToken, [], [{ roles: defaultRole, metadata: {} }])
+        expect(firstToken.status).to.equal(200)
+        const lastToken = await getLastTokenIdRoute(app, authToken)
+        const firstTokenId = lastToken.body.id
+
+        // add new token that will consume
+        const inputs = [firstTokenId]
+        const outputs = [{ roles: defaultRole, metadata: {}, parent_index: 0 }]
+        const secondToken = await postRunProcess(app, authToken, inputs, outputs)
+
+        expect(secondToken.body).to.have.length(1)
+        expect(secondToken.status).to.equal(200)
+
+        const getItemResult = await getItemRoute(app, authToken, { id: firstTokenId + 1 })
+        expect(getItemResult.status).to.equal(200)
+        expect(getItemResult.body.id).to.deep.equal(firstTokenId + 1)
```

Comment:
minor, no need to deep equals on strings on this line and the next assertion

Reply:
we don't, removed various other `deep.equals` as well